src/bindings/nodejs/scripts/bundle.sh

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
is there no node library or some installable script which does the bundling?

does this alter the dependency library names it bundles before RPATHing, e.g. libgeos.so -> libgeos-583a4fe.so? for python packages it's not unusual that systems drops those libraries in the main library dir for many packages and other packages might install a different libgeos.